modules/virt-hardware-os-requirements.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: REFERENCE
+[id="virt-hardware-os-requirements_{context}"]
+= Hardware and operating system requirements
+
+Review the following hardware and operating system requirements for {VirtProductName}.
+
+.Supported platforms
+* On-premise bare metal servers
+* Amazon Web Services bare metal instances
+* IBM Cloud bare metal servers. See link:https://access.redhat.com/articles/6738731[Deploy OpenShift Virtualization on IBM Cloud Bare Metal nodes].
+
```

Comment:
Suggest adding a `+` here to indent the TP note properly

Reply:
Normally that does not work for TP because it's an "include" file. I can try.